src/builder/modules/Generator.res

```diff
@@ -86,6 +86,10 @@ let defineBooleanInput = (id, initial) => {
   model := Builder.defineBooleanInput(model.contents, id, initial)
 }
 
+let defineButtonInput = (id, toRun) => {
```

Comment:
It's conventional to name button functions `onClick` so let's rename `toRun` to `onClick` here.